ios/FluentUI/Command Bar/CommandBarTokens.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class MSFCommandBarTokens: MSFTokensBase {
+    public var backgroundColor: UIColor!
+    public var iconColor: UIColor!
+    
+    public var hoverBackgroundColor: UIColor!
+    public var hoverIconColor: UIColor!
+    
+    public var pressedBackgroundColor: UIColor!
+    public var pressedIconColor: UIColor!
+    
+    public var selectedBackgroundColor: UIColor!
+    public var selectedIconColor: UIColor!
+    
+    public var disabledBackgroundColor: UIColor!
+    public var disabledIconColor: UIColor!
+
+    public var dismissBackgroundColor: UIColor!
```

Comment:
what's dismiss background color for?

Reply:
This is used for the keyboard dismiss!

Reply:
then there is clear mismatch in naming?

Reply:
the demo app is using for keyboard dismiss but the real button type(style) is fixed button isn't it?

Reply:
I believe it means that if the button is not of persist selection type, then it will get the dismissed color properties, sorry if I am getting it confused